jxline-core/src/main/java/cloud/xline/jxline/KV.java

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Is the file directly copied from the other repo?

Reply:
Yes, but it uses our kv types.